applications/solvers/superDeliciousVanilla/superDeliciousVanilla.C

```diff
@@ -123,10 +136,21 @@ int main(int argc, char *argv[])
                 corrIter++;
             }
 
-            // Compute the continuity errors.
-            #include "computeDivergence.H"
+            // If starting without the t^(n-1) data, advance through the remaining
+            // outer iterations without doing anything by calling pimple.loop().
+            if (limitOuterLoop)
+            {
+                while (pimple.loop());
+                Info << endl;
+                break;
+            }
+
+            Info << endl;
         }
 
+        // Compute the continuity errors.
+        #include "computeDivergence.H"
```

Comment:
Given that this is outside of the main pimple loop now, I think that 
https://github.com/mchurchf/SOWFA-6/blob/f/correctRestart/applications/solvers/superDeliciousVanilla/computeDivergence.H#L2
should be something like
```
assert (pimple.finalPimpleIter());
```
or 
```
if (!pimple.finalPimpleIter()) FatalError.exit();
```

Reply:
I thought I had removed the if statement surrounding the code.  Is there an advantage in throwing errors versus just removing the if statement?

Reply:
I'm a fan of sanity checks when they're easy, to help with possible future debugging.

Reply:
In the end, I just got rid of the if statement because I don't want to limit the use of computeDivergence to the end of a time step.  It seems to strong to throw an error for using this piece of code that way.  It may be useful some time to see what it does every outer iteration, but I don't think that is necessary in this code.